.circleci/scripts/label-prs.js

```diff
@@ -0,0 +1,142 @@
+async function runLabelPRsAutomation(context, github) {
```

Comment:
Only things that run in circleci should be in this folder probably, might make sense to add it to the .github folder as it is ran by the github actions.